src/MeetingsSDKAdapter.test.js

```diff
@@ -171,37 +172,46 @@ describe('Meetings SDK Adapter', () => {
     beforeEach(() => {
       meetingSDKAdapter.meetings[meetingID] = {
         ...meeting,
-        localAudio: {
-          getAudioTracks: jest.fn(() => [{enabled: true}]),
-        },
+        localAudio: {},
       };
     });
 
-    test('mutes audio if the the audio track is enabled', () => {
-      meetingSDKAdapter.handleLocalAudio(meetingID);
+    test('mutes audio if the the audio track is enabled', async () => {
+      await meetingSDKAdapter.handleLocalAudio(meetingID);
       expect(mockSDKMeeting.muteAudio).toHaveBeenCalled();
     });
 
-    test('emits the custom event after muting the audio track', () => {
-      meetingSDKAdapter.handleLocalAudio(meetingID);
+    test('disabledLocalAudio property should be defined once the audio track is muted', async () => {
```

Comment:
You should check that `localAudio` is `null` instead, since that's what is on the interface